code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -452,6 +452,9 @@
 	blood_color = "#1F181F"
 	flesh_color = "#AAAAAA"
 	reagent_tag = PROCESS_SYN
+	
+	has_organ = list(
+		)
```

Comment:
shouldn't this be filled with the mechanical organs that do exist--along with a positronic brain?